### ISSUE TITLE
Unhandled Runtime Error TypeError: Cannot read properties of null (reading 'clientHeight')

### ISSUE THREAD

**Issue by mohamadyusuf19** [MetinArslanturk/react-animation-on-scroll#19]
I'm using this library in my NextJS project

Then I got this error:

Unhandled Runtime Error
TypeError: Cannot read properties of null (reading 'clientHeight')

Call Stack
eval
node_modules/react-animation-on-scroll/dist/esm/components/AnimationOnScroll.js (94:0)
eval
node_modules/react-animation-on-scroll/dist/esm/components/AnimationOnScroll.js (155:0)
eval
node_modules/react-animation-on-scroll/dist/esm/components/AnimationOnScroll.js (177:0)
invokeFunc
node_modules/lodash.throttle/index.js (160:0)
leading-edge
node_modules/lodash.throttle/index.js (170:0)
debounced
node_modules/lodash.throttle/index.js (235:0)

Any idea?

**Comment by alex-lechner** [comment]
I have the same error in Next 14.

@MetinArslanturk any hint on what this might be?

**Comment by alex-lechner** [comment]
It seems that in `react-animation-on-scroll/dist/esm/components/AnimationOnScroll.js` there's no null check on line 94

```js
const elementBottom = elementTop + node.current.clientHeight; // 'node.current' is possibly 'null'
```

**Comment by kshitijkumarsaini** [comment]
Any workaround on this?

**Comment by mixartemev** [comment]
> Any workaround on this?

> Any idea?

Yes, I fixed it in my fork: https://github.com/mixartemev/react-animation-on-scroll
